include/ews/ews.hpp

```diff
@@ -7602,6 +7602,545 @@ namespace ews
+        //! <tt>\<ExtendedFieldUri\></tt>.
+        //!
+        //! Identifies the property tag without the type part of the tag.
+        //! The PropertyTag can be represented as either a hexadecimal or a
+        //! short integer. The range between 0x8000 and 0xFFFE represents the
+        //! custom range of properties. When a mailbox database encounters a
+        //! custom property for the first time, it assigns that custom property
+        //! a property tag within the custom property range of 0x8000-0xFFFE.
+        //! A given custom property tag will most likely differ across
+        //! databases.
+        //! Therefore, a custom property request by property tag can return
+        //! different properties on different databases. The use of the
+        //! PropertyTag attribute is prohibited for custom properties.
+        //! Instead, use the PropertySetId attribute and the PropertyName
+        //! or PropertyId attribute.
+        //! If the PropertyTag attribute is used, the
+        //! DistinguishedPropertySetId,
+        //! PropertySetId, PropertyName, and PropertyId attributes cannot be
+        //! used.
+        //! This attribute is optional.
+        class property_tag final
+        {
+        public:
+//! Default constructor
+#ifdef EWS_HAS_DEFAULT_AND_DELETE
+            property_tag() = default;
+#else
+            property_tag(){};
+#endif
+            //! Constructor to create a PropertyTag property
+            explicit property_tag(std::string str) : _str(std::move(str)){};
+            //! Standard destructor
+            ~property_tag(){};
+
+            //! Returns this property as a string.
+            const std::string& str() const EWS_NOEXCEPT { return _str; }
+
+        private:
+            std::string _str;
+        };
+
+#ifdef EWS_HAS_NON_BUGGY_TYPE_TRAITS
+        static_assert(std::is_default_constructible<property_tag>::value, "");
+        static_assert(std::is_copy_constructible<property_tag>::value, "");
+        static_assert(std::is_copy_assignable<property_tag>::value, "");
+        static_assert(std::is_move_constructible<property_tag>::value, "");
+        static_assert(std::is_move_assignable<property_tag>::value, "");
+#endif
+
+        //! \brief Represents the <tt>\<PropertyId\></tt> of an
+        //! <tt>\<ExtendedFieldUri\></tt>.
+        //!
+        //! Identifies an extended property by its dispatch ID. The dispatch ID
+        //! can be identified in either decimal or hexadecimal formats.
+        //! This property must be coupled with either DistinguishedPropertySetId
+        //! or PropertySetId. If this attribute is used, the PropertyName and
+        //! PropertyTag attributes cannot be used. This attribute is optional.
+        class property_id final
+        {
+        public:
+//! Default constructor
+#ifdef EWS_HAS_DEFAULT_AND_DELETE
+            property_id() = default;
+#else
+            property_id(){};
```

Comment:
The semicolon after the closing curly brace is useless and can cause compiler warnings and confusion. Remove here and in all other occurrences in this commit